src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -13,7 +13,9 @@
   <ItemGroup>
     <PackageReference Include="DurableTask.AzureStorage" Version="0.1.0-alpha" />
     <PackageReference Include="Microsoft.AspNet.WebApi.Core" Version="5.2.3" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="2.1.0-beta1-10950" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="2.1.0-beta1-11013" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Logging.ApplicationInsights" Version="2.1.0-beta1-11013" />
```

Comment:
I just realized this -- you actually shouldn't need any package reference to our App Insights package since you don't touch it. You only care about ILogger/ILoggerFactory and trust Functions to set up the App Insights logging. That should simplify your dependencies.